Python/pythonrun.c

```diff
@@ -1538,24 +1538,21 @@ _Py_SourceAsString(PyObject *cmd, const char *funcname, const char *what, PyComp
 /*
  * Return non-zero when we run out of memory on the stack; zero otherwise.
  */
-int
-PyOS_CheckStack(void)
+int PyOS_CheckStack(void)
```

Comment:
Nitpick: please revert the style change
```suggestion
int
PyOS_CheckStack(void)
```